samples/msal-react-sample/src/index.tsx

```diff
@@ -12,7 +12,7 @@ import { Configuration } from "@azure/msal-browser";
 const configuration: Configuration = {
```

Comment:
can we separate the config file as other samples to have more flexibility in testing?

Reply:
God point. I've split the `Configuration` into a [`config.ts`](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/156db446b880dae69f9c9f68ebe6e4425af10ded/samples/msal-react-sample/src/config.ts) file and also moved the values into the [.env](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/156db446b880dae69f9c9f68ebe6e4425af10ded/samples/msal-react-sample/.env) file.